scripts/onebranch/post-build.ps1

```diff
@@ -7,4 +7,52 @@ $scriptPath = Split-Path -Parent $MyInvocation.MyCommand.Definition
 # Change the parent directory for the script directory
 Set-Location $scriptPath\..\..
 
-# Add option to build MSI here
\ No newline at end of file
+# Copyright (c) eBPF for Windows contributors
+# SPDX-License-Identifier: MIT
```

Comment:
Looks like a duplicate copyright/SPDX ended up in here - it could be removed

Reply:
And the definition of `$scriptPath` and the `Set-Location` already happen just above this, so the duplicate copies just below can be removed

Reply:
Fixed.